syntax = "proto3";

message FaceRecognitionRequest {
    string username = 1;
    string media_content = 2;
    string media_type = 3;
}

message FaceRecognitionResponse {
    string message = 1;
    string data = 2;
    string username = 3;
    string verification = 4;
}

service FaceRecognitionService {
    rpc FaceRecognition (FaceRecognitionRequest) returns (FaceRecognitionResponse);
}